classes/class3/Homework/HomeworkApp/routes/index.js

```diff
@@ -0,0 +1,60 @@
+var home = function(req, res) {
+	res.render("home", {
+		"classes": [
+			"Olin.js",
+			"other class 1",
+			"other class 2",
+			"other class 3"
+		]
+	});
+};
+
+var deleteCat = function(req, res) {
```

Comment:
Unused code. You should move the inline routes from app.js into these functions and delete the ones you aren't using (`home`)